packages/interface/src/api/activity-certificate/endpoint/apiAcf004.ts

```diff
@@ -0,0 +1,67 @@
+import { HttpStatusCode } from "axios";
+import { z } from "zod";
+
+import { ActivityCertificateOrderStatusEnum } from "@sparcs-clubs/interface/common/enum/activityCertificate.enum";
+import { zKrPhoneNumber } from "@sparcs-clubs/interface/common/type/phoneNumber.type";
+
+/**
+ * @version v0.1
+ * @description 신청한 활동확인서 발급 신청 정보를 가져옵니다
+ */
+
+const url = (id: number) =>
+  `/student/activity-certificates/activity-certificate/${id}`;
+const method = "GET";
+
+const requestParam = z.object({
+  id: z.number().int().positive(),
+});
+
+const requestQuery = z.object({});
+
+const requestBody = z.object({});
+
+const responseBodyMap = {
+  [HttpStatusCode.Ok]: z.object({
+    id: z.number().int(),
+    clubId: z.number().int(),
+    studentNumber: z.number().int(),
+    studentPhoneNumber: zKrPhoneNumber,
+    issuedNumber: z.number().int().positive(),
+    statusEnum: z.nativeEnum(ActivityCertificateOrderStatusEnum),
+    items: z.array(
+      z.object({
+        id: z.number().int(),
+        startMonth: z.date(),
+        endMonth: z.date(),
```

Comment:
💊 아마 date 관련 zod 객체들이 coerce 없이 파싱이 안되었던 것으로 기억합니다. 
```suggestion
        startMonth: z.coerce.date(),
        endMonth: z.coerce.date(),
```